app/controllers/rental_items_controller.rb

```diff
@@ -28,6 +30,9 @@ def create
 
     respond_to do |format|
       if @rental_item.save
+        params[:rental_item_images]['item_image'].each do |a|
+          @rental_item_image = @rental_item.rental_item_images.create!(:item_image => a)
+        end
```

Comment:
Not sure how this library works but seems weird to assign a controller variable here in the loop, as it will just override the value each time. Unless if you just want the last value assigned for some reason.